packages/compartment-mapper/test/test-main.js

```diff
@@ -92,7 +92,7 @@ test('no dev dependencies', async t => {
       });
     },
     {
-      message: /Cannot find external module "typecommon"/,
+      message: /Cannot find external module "typemodule"/,
```

Comment:
I doublechecked - it's just a matter of different order. If I remove other dev deps and their mentions, I do get the typecommon package error again.

Reply:
Sounded great at first, but then I thought - until we have much tighter feature coverage for cjs support, a test that may fail on changes in order of things happening could be valuable. It took me 5 minutes to figure out the behavior is still correct and I got a nice signal from it.

I'll add a TOO comment to relax this assertion later, if you don't mind.